Makefile

```diff
@@ -20,11 +28,42 @@ run: build
 clean:
 	rm -f $(APP_NAME)
 	rm -f model-runner.sock
+	rm -rf $(MODELS_PATH)
+
+# Run tests
+test:
+	go test -v ./...
+
+# Build Docker image
+docker-build:
+	docker build --platform linux/amd64 \
```

Comment:
`--platform linux/amd64` because there's not a llama.cpp build for linux/arm64 yet, right?

Reply:
@doringeman yes